start-aws.py

```diff
@@ -0,0 +1,62 @@
+import os
+import time
+
+import boto.ec2
+
+
+# AWS region
+aws_region = 'us-west-2'
+
+# Name of security group exposing ports for Buildmaster
+security_group = 'Buildbot staging'
+
+# Key name for EC2 host
+key_name = 'hybrid-master'
+
+
+ec2_image = {
+    'Fedora-x86_64-20-20140407-sda': 'ami-cc8de6fc'
+}
+
+conn = boto.ec2.connect_to_region(aws_region)
+
+# Buildbot master requires more than standard 4Gb disk.  Set the EBS
```

Comment:
This is unclear.  The first sentence should be reworded similar to "Buildbot master requires a 4Gb disk which is larger than the default". Else it might be understood (as it was by me initially) to mean "Buildbot master requires more than the standard 4Gb disk"